ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -17,8 +17,6 @@ const PredictionValue = styled.span`
   text-transform: uppercase;
 `
 
-const NUM_TO_SHOW_ABOVE_THE_FOLD = 6 // how many predictors to show immediately
```

Comment:
I added back in the `NUM_TO_SHOW_ABOVE_THE_FOLD` behavior and implemented the sorting approach instead. I added `displaySortOrder` as a new `PREDICTOR_FIELDS` option and sorted the filtered fields according to that value.

Reply:
is there a reason to add `displaySortOrder` as a field, instead of just using the order of `PREDICTOR_FIELDS` as the display order?

Reply:
I added the sort order as a field option because it is more explicit to a
reader that order of the predictors matters somewhere, and the significance
of ordering isn’t hidden in the implementation of the PREDICTOR_FIELDS as
an array. If another field is added anywhere in the array the same order
remains.

However, this means right now that the displaySortOrder values match the
order of
PREDICTOR_FIELDS so the sorting logic I added is redundant.

I’m flexible and open to removing  displaySortOrder if you think
PREDICTOR_FIELDS won’t need to be changed too much in the future.

Reply:
My general assumption is that arrays of things to display are ordered lists, and that for things displayed to users order is generally important. So to me, its implied that the order of `PREDICTOR_FIELDS` is meant to be ordered, but thats clearly not universal. I don't really like the idea of needing to maintain the additional `displaySortOrder` since I don't really see an advantage to allowing the list to have a different order than the display, but maybe renaming the array `ORDERED_PREDICTOR_FIELDS` or `DISPLAY_ORDER_PREDICTOR_FIELDS` or something else that makes it clear that the order is important will make it explicit that the order is being used without adding any additional fields to maintain